group_vars/prosody_prod/vars.yml

```diff
@@ -0,0 +1,11 @@
+###
+# Production specific variables for ppa-django
+###
+---
+
+
+# solr settings
+# NOTE: should be moved somewhere common; zk is unique but solr vars used for old deploys
+zk_host: "lib-zk-staging1:2181,lib-zk-staging2:2181,lib-zk-staging3:2181/solr8"
+solr_url: "http://lib-solr8-staging.princeton.edu:8983/solr/"
+solr_server: "{{ groups['solr_prod'][0] }}"
```

Comment:
copy whatever we're using in shxco production variables for now (but please keep the comment)

RSK todo – figure out where/how to move these to common variables to reuse across playbooks